src/__tests__/Tooltip-test.js

```diff
@@ -86,4 +86,90 @@ describe('Tooltip', () => {
         expect(label.props.className)
             .toExclude('mdl-tooltip--large');
     });
+
+    it('should not render any tooltip positioning css if unspecified', () => {
+        const element = (
+            <Tooltip label="my tooltip">
+                <Icon name="add" />
+            </Tooltip>
+        );
+        const output = render(element);
+
+        const label = output.props.children[1].props.children;
+
+        expect(label.props.className)
+            .toExclude('mdl-tooltip--left')
+            .toExclude('mdl-tooltip--right')
+            .toExclude('mdl-tooltip--top')
+            .toExclude('mdl-tooltip--bottom');
+    });
+
+    it('should render tooltip left css if specified', () => {
+        const element = (
+            <Tooltip label="my tooltip" position="left">
+                <Icon name="add" />
+            </Tooltip>
+        );
+        const output = render(element);
+
+        const label = output.props.children[1].props.children;
+
+        expect(label.props.className)
+            .toInclude('mdl-tooltip--left');
+    });
+
+    it('should render tooltip right css if specified', () => {
+        const element = (
+            <Tooltip label="my tooltip" position="right">
+                <Icon name="add" />
+            </Tooltip>
+        );
+        const output = render(element);
+
+        const label = output.props.children[1].props.children;
+
+        expect(label.props.className)
+            .toInclude('mdl-tooltip--right');
+    });
+
+    it('should render tooltip top css if specified', () => {
+        const element = (
+            <Tooltip label="my tooltip" position="top">
+                <Icon name="add" />
+            </Tooltip>
+        );
+        const output = render(element);
+
+        const label = output.props.children[1].props.children;
+
+        expect(label.props.className)
+            .toInclude('mdl-tooltip--top');
+    });
+
+    it('should render tooltip bottom css if specified', () => {
+        const element = (
+            <Tooltip label="my tooltip" position="bottom">
+                <Icon name="add" />
+            </Tooltip>
+        );
+        const output = render(element);
+
+        const label = output.props.children[1].props.children;
+
+        expect(label.props.className)
+            .toInclude('mdl-tooltip--bottom');
+    });
+
+    it('should display console warning if specified position is invalid', () => {
```

Comment:
Oh yeah, I'm a bit confused about testing warning in the console. I saw similar scenarios in other components (for example [Tabs-test.js:85](https://github.com/tleunen/react-mdl/blob/master/src/Tabs/__tests__/Tabs-test.js#L85), but it simply render the element without expecting anything. I'll just remove it then.